account_payment_return_import/tests/test_import_base.py

```diff
@@ -27,62 +27,9 @@ def setUpClass(cls):
                 "name": "Test Bank Journal",
                 "code": "BANK",
                 "type": "bank",
-                "update_posted": True,
                 "bank_account_id": cls.acc_bank.id,
             }
         )
-        cls.journal.bank_account_id = cls.acc_bank
```

Comment:
Why removing all this stuff? This is probably needed for subsequent modules.

Reply:
I hesitated to delete it, and yet I deleted it. Next time I have doubts about deleting something or not in a base module I wont delete anything, haha.